.github/workflows/phpunit.yml

```diff
@@ -46,15 +47,15 @@ jobs:
 
       - name: "Install lowest dependencies"
         if: ${{ matrix.dependencies == 'lowest' }}
-        run: "composer update --prefer-lowest --no-interaction --no-progress"
+        run: "composer update --prefer-lowest --no-interaction --no-progress --ignore-platform-req=php"
```

Comment:
This will be trouble, since many packages could be updated to ancient releases.

To keep builds stable, I think we should now try one last thing:

 * Require PHP 8 (only)
 * Remove all `--ignore-platform-req=`

I suggest doing it in a separate commit, so we can evaluate whether to do it here or to defer to a next minor.

Reply:
done, though now we have failing CI 🤔

Reply:
Alrighty, no problem.

I'll take over the merge once at a computer 👍